go.mod

```diff
@@ -85,3 +86,5 @@ require (
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20231127180814-3a041ad873d4 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+exclude github.com/knadh/koanf v1.5.0
```

Comment:
opentelemetry depends on this version so it is added as an indirect one. The problem is that some subpackages names collide with the ones in the v2 so it fails to build because it doesn't know which one to import